io.github.cosmicUtils.cosmicAppletMinimon.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "io.github.cosmicUtils.cosmicAppletMinimon",
```

Comment:
the id does not follow the requirements

Reply:
It should be `io.github.cosmic_utils.minimon-applet`, you have to update it in other places it is used.

Reply:
Got it, thanks. I'll do that.

Reply:
Will be fixed/ready when [this](https://github.com/cosmic-utils/minimon-applet/pull/59) is merged. I'll let you know.